dev/custom-pages/homepage.scss

```diff
@@ -270,8 +269,29 @@ $color-linkedin: #0077b5;
     }
 
     .padded-bottom {
-      padding-bottom: 70px;
+      padding-bottom: 47px;
     }
+
+    @media (max-width: $screen-md-max) {
```

Comment:
I'm fairly certain these css changes shouldn't even be in here... They're completely unrelated to the change set. I'm not quite sure how they got in here.